core/src/main/java/org/apache/iceberg/rest/RESTCatalogProperties.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.rest;
+
+/**
+ * Catalog properties which are specific to the RESTCatalog, which can be used in conjunction with
+ * {@link org.apache.iceberg.CatalogProperties}
+ */
+public class RESTCatalogProperties {
+
+  private RESTCatalogProperties() {
+  }
+
+  /**
+   * A Bearer authorization token which will be used to authenticate requests with the server.
+   */
+  public static final String AUTH_TOKEN = "token";
```

Comment:
Do we want this to start with `rest.`, similar to the way that `jdbc` catalog prefaces its configs?

Reply:
Yeah I don't think so either. Should be easy to add if need be.